css/main.scss

```diff
@@ -86,5 +86,6 @@ $flagsImagePath: "../images/";
 @import 'avatar';
 @import 'promotional-footer';
 @import 'chrome-extension-banner';
+@import 'close';
```

Comment:
Why make the close page depend on all.css? You can inline the style in the HTML page itself. Or do you need anything else from the all.css file?

Reply:
fair enough

Reply:
done